### PR TITLE
Strain design

### PR DESCRIPTION
**Pull request by AlbaLopez**

### CODE REVIEW

File: src/environments/environment.prod.ts [hunk 1]
```diff
@@ -25,14 +25,12 @@ export const environment: Configuration = {
...
+    design_storage: 'https://api-staging.dd-decaf.eu/design-storage',
   },
   GA: {
     trackingID: 'UA-106144097-2',
   },
-  sentry: {
-    DSN: 'https://4f2d0b394b5a4215806fc38ae2424502@sentry.io/1233163',
-    release: 'SENTRY_PROJECT',
-  },
+  sentry: null,
```
Comment (Midnighter) [inline src/environments/environment.prod.ts]: I believe sentry information should remain in the production environment.

File: src/app/app-models/components/add-model/add-model.component.html [hunk 2]
```diff
@@ -123,10 +123,11 @@ <h3 class="title">Add new organism</h3>
         <input matInput cdkFocusInitial #neworganism/>
       </mat-form-field>
       <mat-form-field>
-        <mat-label class="title">Project*</mat-label>
+        <mat-label class="title">Project* {{selectedProject}}</mat-label>
```
Comment (Midnighter) [inline src/app/app-models/components/add-model/add-model.component.html]: Remove ` {{selectedProject}}`

File: src/app/services/ninja-service.ts [hunk 3]
```diff
@@ -56,6 +57,16 @@ export class NinjaService {
   }
 
   getPredictions(): Observable<Job[]> {
-    return this.http.get<Job[]>(`${environment.apis.metabolic_ninja}/predictions`);
+    return this.http.get<Job[]>(`${environment.apis.metabolic_ninja}/predictions`).pipe(map((predictions: Job[]) =>
+      this.processPredictions(predictions)));
+  }
+
+  processPredictions(predictions: Job[]): Job[] {
```
Comment (Midnighter) [inline src/app/services/ninja-service.ts:64]: Double check for existing model input.

**Review by Midnighter — COMMENTED** [review]
@AlbaLopez and I discussed the PR in person. We found a few parts that need to be fixed otherwise it looks good!

File: src/app/services/ninja-service.ts (hunk 3 above)
Comment (AlbaLopez) [reply src/app/services/ninja-service.ts:64]: Checked, and it is necessary.

**Pull request merged**